Makefile.inc

```diff
@@ -19,7 +19,11 @@
 # nor periods (problems on xtensa)
 
 VERSION := V05_00_work_in_progress_2021_01_04
-WSL := $(strip $(shell grep -qE "(microsoft|Microsoft|WSL)" /proc/version @> /dev/null && printf true))
+ifeq ($(WSL),true)
+   WSL := $(strip $(shell grep -qE "(microsoft|Microsoft|WSL)" /proc/version @> /dev/null && printf true))
```

Comment:
Dit wordt op deze manier nooit uitgevoerd, tenzij `$WSL` al op true staat - maar in dat geval heeft het herdefiniëren weinig zin, toch?